### PR TITLE
TCOMP-568: fix cyclic deps

### PR DESCRIPTION
**Pull request by sgandon**
**What is the current behavior?** (You can also link to an open issue here)
A new component for OSGI unit test was added but depended on *component-common* but should not caus it creates a cyclic deps. We did not see the issue cause is was depending on an old version.

**What is the new behavior?**
The dependency has been removed and then reverted the revert commit.

**Please check if the PR fulfills these requirements**

- [x] The commit message follows Talend standard
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The code coverage on new code >75%
- [ ] The new code does not introduce new technical issues (sonar / eslint)

**What kind of change does this PR introduce?**

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [x] Build / CI related changes
- [ ] Other... Please describe:

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

### CODE REVIEW

File: core/components-osgi-test/pom.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.talend.components</groupId>
+    <artifactId>components-osgi-test-aggregator</artifactId>
+    <version>0.19.0-SNAPSHOT</version>
```
Comment (pteyssier) [inline core/components-osgi-test/pom.xml]: it's now version 0.20.0-SNAPSHOT

Comment (sgandon) [inline core/components-osgi-test/pom.xml]: bien vu l'aveugle !

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/590/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/590/)

**Comment by pteyssier** [comment]
MLTG

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/591/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/591/)

**Pull request merged**